joomla/joomla-cms
# [3.10] Add a warning for non-modern router setups

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/35013

### Summary of Changes

Add a warning for non-modern router setups

### Testing Instructions

Install 3.10-rc2
- apply this patch
- point to a Joomla 4 updater: https://update.joomla.org/core/nightlies/next_major_list.xml
- check is red
- setup the modern router setting for all listed extensions
- check is gree

### Actual result BEFORE applying this Pull Request

There was no check for the modern router in the pre upgrade checker

### Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/2596554/129263558-8952d9a3-7163-4da7-ae2a-c63c50ab2c98.png)

### Documentation Changes Required

-[ ] https://docs.joomla.org/Pre-Update_Check -> Has to be updated
-[ ] Add the router stuff to the migration docs

TTs have to be informed about the added language strings.

cc @joomla/cms-release-team @HLeithner @bembelimen
